### DISCUSSION TITLE
KoboldCPP-v1.79.1.yr1-ROCm missing gfx1100 (7900XTX) files

### DISCUSSION THREAD

**Discussion by daMustermann** [YellowRoseCx/koboldcpp-rocm#86] [General]
The newest release has no gfx1100 files included. I copied them from the yr0 version and it works. I guess that's why it got so much smaller in filesize.
Also, when you extract KoboldCPP-ROCM from the GUI-Extras menu, it is missing the koboldcpp.py - only tried it with yr1

**Comment by Tatalebuj** [comment]
Thanks for posting this, but I'm a bit too non-technical to understand how to resolve this.  I'm currently running 175, but would like to upgrade to 179.  I'm not sure where (and which) the "gfx1100 files" are, nor do I know what "yr0" means - though I'm assuming it's a version control number.  I just don't see any folders with that name that would let me copy the contents.  I'm running Windows 11 pro and was hoping you could point me in the right direction to get this upgrade fully deployed?  If you have time, I would greatly appreciate it.  Cheers,

**Reply by daMustermann** [reply]
Just download the new 1.80 release, it is fixed now. The gfx1100 is a codename for the Radeon 7900 series and some workstation GPUs. The files for this line of GPU were missing in the last release before version 1.80. Now they are back in and everything should work as good as the older versions.

https://github.com/YellowRoseCx/koboldcpp-rocm/releases/tag/v1.80.3.yr0-ROCm

**Reply by Tatalebuj** [reply]
Thank yoU!!
